fantasybot/strategy/history.py

```
@@ -0,0 +1,249 @@
+
+        while buys and sells:
+            b = buys.pop(0)
+            s = sells.pop(0)
+            diff = s["amount"] - b["amount"]
+            roi = (diff / b["amount"]) * 100 if b["amount"] else 0
+
+            try:
+                d_buy = datetime.strptime(b["date"], "%Y-%m-%d")
+                d_sell = datetime.strptime(s["date"], "%Y-%m-%d")
+                days = (d_sell - d_buy).days
+            except Exception:
```

Ruff (0.16.1)

[warning] Do not catch blind exception: `Exception`

(BLE001)